libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -985,6 +985,7 @@ IotMqttError_t IotMqtt_Connect( const IotMqttNetworkInfo_t * pNetworkInfo,
     int8_t contextIndex = -1;
     bool subscriptionMutexCreated = false;
     MQTTStatus_t managedMqttStatus = MQTTBadParameter;
+    bool contextMutex = false;
```

Comment:
A variable named `contextMutex` sounds like an instance of type `Mutex` rather than a flag indicating that a mutex has been created successfully.  Let's rename this to `contextMutexCreated`, similar to the flag named `networkCreated`.

Reply:
Thanks. Addressed it.